stories/topics/con-aws-application-architecture.adoc

```diff
@@ -22,6 +22,9 @@ This podman configuration is itself managed by systemd.
 SELinux is enabled on our instances and is supported down to the container level. 
 Each {ControllerName} instance is configured as an {PlatformNameShort} hybrid node by default to simplify the scale up and out procedures.
 
+You can't restart virtual machines, instead virtual machines must be replaced. You can't do any type of configuration on virtual machines, because this configuration will be blown away when virtual machines are replaced which happens by design. For example, this always happen on upgrade. 
```

Comment:
You cannot restart.

You cannot configure virtual machines, because any configuration is lost when virtual machines are replaced. 
This always happens on upgrade.